### ISSUE TITLE
Cannot create property 'context' on string 'Error: Cannot find module '~\webconfig      ailwind.config.js'

### ISSUE THREAD

**Issue by reachej** [redwoodjs/redwood#6255]
### What's not working?

If anyone is getting the subject error when trying to generate a cell, change the line in the prettier.config.js

from:
tailwindConfig: './web\config\tailwind.config.js',

to:
tailwindConfig: '.\\web\\config\\tailwind.config.js',

For some reason on windows the logs are showing forward and back slashes in the logs too:

  √ Generating page files...
    √ Successfully wrote file `./web\src\pages\HomePage\HomePage.stories.js`
    √ Successfully wrote file `./web\src\pages\HomePage\HomePage.test.js`
    √ Successfully wrote file `./web\src\pages\HomePage\HomePage.js`

### How do we reproduce the bug?

yarn create redwood-app ./blog

  √ Generating page files...
    √ Successfully wrote file `./web\src\pages\HomePage\HomePage.stories.js`
    √ Successfully wrote file `./web\src\pages\HomePage\HomePage.test.js`
    √ Successfully wrote file `./web\src\pages\HomePage\HomePage.js`
  √ Updating routes file...
  √ Generating types...
  √ One more thing.

yarn rw setup ui tailwindcss

yarn rw g cell teamMembers

PS PathIsHere\blog> yarn rw g cell teamMembers
  \ Generating cell files...
    Generating types ...
Cannot create property 'context' on string 'Error: Cannot find module 'PathIsHere\blog\webconfig ailwind.config.js'
Require stack:
- PathIsHere\blog\node_modules\prettier\noop.js

### What's your environment? (If it applies)

```shell
System:
    OS: Windows 10 10.0.22000
  Binaries:
    Node: 16.16.0 - ~\AppData\Local\Temp\xfs-add38609\node.CMD
    Yarn: 3.2.1 - ~\AppData\Local\Temp\xfs-add38609\yarn.CMD
  Browsers:
    Chrome: 104.0.5112.82
    Edge: Spartan (44.22000.120.0), Chromium (104.0.1293.63)
  npmPackages:
    @redwoodjs/core: 2.2.0 => 2.2.0
```

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by Tobbe** [comment]
@reachej Thanks for your bug report. Do you think this is the same issue? https://github.com/redwoodjs/redwood/pull/6203

**Comment by reachej** [comment]
Yes.  However, I think the more real problem is whatever is causing redwood to receive a backslash instead of forward slash when running on windows (that is very likely why the generated config is wrong) / there shouldn't be mixtures of forward slashes and backslashes in logs like the below:

For example:

Instead of

√ Successfully wrote file ./web\src\pages\HomePage\HomePage.stories.js
√ Successfully wrote file ./web\src\pages\HomePage\HomePage.test.js
√ Successfully wrote file ./web\src\pages\HomePage\HomePage.js

one should get

√ Successfully wrote file ./web/src/pages/HomePage/HomePage.stories.js
√ Successfully wrote file ./web/src/pages/HomePage/HomePage.test.js
√ Successfully wrote file ./web/src/pages/HomePage/HomePage.js

**Comment by Tobbe** [comment]
We do `process.cwd()` to get the base path. On Windows node will return something like `'C:\\Users\\tobbe\\dev\\redwood\\my-redwood-app'`. So that's what's causing redwood to receive a backslash. That's just the way things work on Windows. Not much we can do about that.

The reason for the mix of / and \ is this:
```
task.title = `Successfully wrote file \`./${path.relative(base, target)}\``
```

As you can see the first slash is hardcoded. We should probably use `path.sep` instead. That'd be a pretty big change though, because I bet we have a lot of hardcoded slashes like that.

@reachej Do you agree with my conclusions? And if you so, do you want to try to fix it?

**Comment by Tobbe** [comment]
Found this issue related to the PR I linked earlier https://github.com/redwoodjs/redwood/issues/6194

It has some more discussion about the paths. And also this comment https://github.com/redwoodjs/redwood/pull/6203/files/75145d25684452f37c0a2b5f4b27b17f66794de7#r943605422
